test/gui/shared/scripts/helpers/api/webdav_helper.py

```diff
@@ -64,3 +64,11 @@ def create_file(user, file_name, contents):
     assert (
         response.status_code == 201
     ), f"Could not create file '{file_name}' for user {user}"
+
+
+def delete_folder(user, folder_name):
+    url = get_resource_path(user, folder_name)
+    response = request.delete(url, user=user)
+    assert (
+        response.status_code == 204
+    ), f"Could not delete folder '{folder_name}' for user {user}"
```

Comment:
It can be used for both files and folders
```suggestion
def delete_resource(user, resource):
    url = get_resource_path(user, resource)
    response = request.delete(url, user=user)
    assert (
        response.status_code == 204
    ), f"Could not delete resource '{folder_name}'"
```